main/integration-tests/__test__/api-tests/appstream-egress-enabled/data-egress/get-data-egress.test.js

```diff
@@ -70,6 +70,7 @@ describe('Create URL scenarios', () => {
             StorageClass: 'STANDARD',
             projectId: 'TRE-Project',
             workspaceId: 'ea8e5286-45ca-402a-8c98-7d4495f646e2',
+            ChecksumAlgorithm: [], // new attribute introduced in Feb 2022 https://aws.amazon.com/blogs/aws/new-additional-checksum-algorithms-for-amazon-s3/
```

Comment:
I don't think the comment is necessary

Reply:
Sanket suggested it so people that look at this test later know why it was needed.